packages/block-library/src/cover/block.json

```diff
@@ -19,9 +19,6 @@
 		},
 		"alt": {
```

Comment:
I wonder if the name of the attribute is reflective of what it does now :)

Reply:
Maybe the name could be "accessibleText". I'm not sure that is clearer.